### ISSUE TITLE
`RedisStore` doesn't implement `std::fmt::Debug`

### ISSUE THREAD

**Issue by Fyko** [maxcountryman/tower-sessions#29]
When trying to use a RedisStore inside a CachingSessionStore, it fails, because `RedisStore` doesn't implement debug.
## Code
```rs
let config = RedisConfig::from_url(&CONFIG.redis_url)?;
let perf = PerformanceConfig::default();
let policy = ReconnectPolicy::default();
let client = RedisClient::new(config, Some(perf), Some(policy));

let redis_conn = client.connect();
client.wait_for_connect().await?;

let moka_store = MokaStore::new(Some(2000));
let redis_store = RedisStore::new(client);
let caching_store = CachingSessionStore::new(moka_store, redis_store);

let session_layer = ServiceBuilder::new()
	.layer(HandleErrorLayer::new(|_: BoxError| async {
		StatusCode::BAD_REQUEST
	}))
	.layer(
		SessionManagerLayer::new(caching_store)
			.with_secure(true)
			.with_max_age(time::Duration::seconds(10)),
	);
```
## Error
```rs
error[E0277]: `RedisStore` doesn't implement `std::fmt::Debug`
   --> apps/appealsy-api/src/lib.rs:49:38
    |
49  |             SessionManagerLayer::new(caching_store)
    |             ------------------------ ^^^^^^^^^^^^^ `RedisStore` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
    |             |
    |             required by a bound introduced by this call
    |
    = help: the trait `std::fmt::Debug` is not implemented for `RedisStore`
    = help: the trait `SessionStore` is implemented for `CachingSessionStore<Cache, Store>`
    = note: required for `CachingSessionStore<MokaStore, RedisStore>` to implement `SessionStore`
note: required by a bound in `SessionManagerLayer::<Store>::new`
   --> /home/carter/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tower-sessions-0.2.1/src/service.rs:255:13
    |
255 | impl<Store: SessionStore> SessionManagerLayer<Store> {
    |             ^^^^^^^^^^^^ required by this bound in `SessionManagerLayer::<Store>::new`
...
267 |     pub fn new(session_store: Store) -> Self {
    |            --- required by a bound in this associated function

error[E0599]: the method `with_secure` exists for struct `SessionManagerLayer<CachingSessionStore<MokaStore, RedisStore>>`, but its trait bounds were not satisfied
  --> apps/appealsy-api/src/lib.rs:50:18
   |
49 | /             SessionManagerLayer::new(caching_store)
50 | |                 .with_secure(true)
   | |                 -^^^^^^^^^^^ method cannot be called due to unsatisfied trait bounds
   | |_________________|
   | 
   |
  ::: /home/carter/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tower-sessions-0.2.1/src/session_store.rs:59:1
   |
59 |   pub struct CachingSessionStore<Cache: SessionStore, Store: SessionStore> {
   |   ------------------------------------------------------------------------ doesn't satisfy `_: SessionStore`
   |
   = note: the following trait bounds were not satisfied:
           `CachingSessionStore<MokaStore, RedisStore>: SessionStore`

error[E0277]: `RedisStore` doesn't implement `std::fmt::Debug`
   --> apps/appealsy-api/src/lib.rs:49:13
    |
49  |             SessionManagerLayer::new(caching_store)
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `RedisStore` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
    |
    = help: the trait `std::fmt::Debug` is not implemented for `RedisStore`
    = help: the trait `SessionStore` is implemented for `CachingSessionStore<Cache, Store>`
    = note: required for `CachingSessionStore<MokaStore, RedisStore>` to implement `SessionStore`
note: required by a bound in `SessionManagerLayer`
   --> /home/carter/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tower-sessions-0.2.1/src/service.rs:154:39
    |
154 | pub struct SessionManagerLayer<Store: SessionStore> {
    |                                       ^^^^^^^^^^^^ required by this bound in `SessionManagerLayer`
```

**Comment by maxcountryman** [comment]
I think this should be fixed in https://github.com/maxcountryman/tower-sessions/commit/2cee45b3756b11a37e6f366925d9aa70a5ca3db1.

Happy to cut another release if that helps.

**Comment by maxcountryman** [comment]
I published `0.2.2` which includes that commit. If you have a chance to try it, let me know if that works for you.
